bindings/java/c/host.c

```diff
@@ -35,80 +35,80 @@ static jbyteArray CopyDataToJava(JNIEnv* jenv, const void* ptr, size_t size)
 
 static bool account_exists_fn(struct evmc_host_context* context, const evmc_address* address)
 {
+#ifdef DEBUG
+    printf("Entering account_exists_fn\n");
+#endif
     bool result = false;
     const char java_method_name[] = "account_exists";
     const char java_method_signature[] = "(I[B)I";
     assert(context != NULL);
     JNIEnv* jenv = attach();
-    if (jenv != NULL)
```

Comment:
Can you remind me why all the jenv null checks were removed?

Reply:
Nvmd.  I suppose the attach() method's assert handles the case where AttachCurrentThread fails.

Reply:
cmake fails to compile otherwise.

Reply:
What is the CMake issue?